src/Services/Masa.Scheduler.Services.Server/Domain/EventHandler/NotifyTaskRunResultDomainEventHandler.cs

```diff
@@ -9,13 +9,19 @@ public class NotifyTaskRunResultDomainEventHandler
     private readonly SchedulerDbContext _dbContext;
     private readonly IRepository<SchedulerJob> _schedulerJobRepository;
     private readonly IHubContext<NotificationsHub> _hubContext;
+    private readonly IDistributedCacheClient _distributedCacheClient;
+    private readonly QuartzUtils _quartzUtils;
+    private readonly IIntegrationEventBus _eventBus;
 
-    public NotifyTaskRunResultDomainEventHandler(IRepository<SchedulerTask> schedulerTaskRepository, SchedulerDbContext dbContext, IRepository<SchedulerJob> schedulerJobRepository, IHubContext<NotificationsHub> hubContext)
+    public NotifyTaskRunResultDomainEventHandler(IRepository<SchedulerTask> schedulerTaskRepository, SchedulerDbContext dbContext, IRepository<SchedulerJob> schedulerJobRepository, IHubContext<NotificationsHub> hubContext, IDistributedCacheClient distributedCacheClient, QuartzUtils quartzUtils, IIntegrationEventBus eventBus)
```

Comment:
to many params can format
![image](https://user-images.githubusercontent.com/6133287/175203245-17cfd3fb-2f7e-48e9-bfc1-df81bb9d20aa.png)